flocker/node/agents/gce.py

```diff
@@ -230,6 +230,55 @@ def finished_operation_result():
         return final_operation
 
 
+def wait_for_operation_async(reactor, compute, operation, timeout_steps):
+    """
+    Fires a deferred once a GCE operation is complete, or timeout passes.
+
+    This function will poll the operation until it reaches state 'DONE' or
+    times out, and then returns the final operation resource dict.
+
+    :param reactor: The twisted ``IReactorTime`` provider to use to schedule
+        delays.
+    :param compute: The GCE compute python API object.
+    :param operation: A dict representing a pending GCE operation resource.
+        This can be either a zone or a global operation.
+    :param timeout_steps: Iterable of times in seconds to wait until timing out
+        the operation.
+
+    :returns Deferred: A Deferred firing with the concluded GCE operation
+        resource or calling its errback operation times out.
```

Comment:
typo "if it times out" (but not worth fixing this if you have to re-run all the builds)